utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -323,7 +323,19 @@ int main(int argc, char **argv)
     double wallTime;
     string testCaseName;
 
-    // Initialize buffers for any reductionType functions (testCase 87 - tensor_sum alone cannot return final sum as 8u/8s due to overflow. 8u inputs return 64u sums, 8s inputs return 64s sums)
+    if(testCase == 82 && imagesMixed)
```

Comment:
This PR has nothing to do with RICAP. Pls revert this L326-L338 removal and additions

Reply:
reverted

Reply:
@HazarathKumarM No, Tensor_hip file still has it

Reply:
Resolved